sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_custom_operation.asciidoc

```diff
@@ -0,0 +1,184 @@
+= sycl_ext_oneapi_enqueue_custom_operation
```

Comment:
Why is this called "custom operation"?  That seems too generic.  It seems like this extension only lets you submit tasks that use native interop.  Maybe "enqueue_native_operation"?

Reply:
This is in line with the AdaptiveCpp extension. It was decided that we should rename the ext so that it has `oneapi` in the name, making it not depend on a second implementation. The naming of `enqueue_custom_operation` was chosen so that it's clear that it's the oneapi version of the AdaptiveCpp extension.

Reply:
So for the vendor extension we propose to maintain the AdaptiveCpp name so it's clear that it's equivalent to this, but then for the KHR we would propose a different name. My current thinking is something like `enqueue_native_command` or `enqueue_interop_command` but we can bikeshed that.

Reply:
Let's decide a good name now.  If we publish this as a vendor extension, our customers will be using it for a while.  We should have a good name for them.  I'm not as concerned about keeping the same name as ACPP.

Reply:
I like `enqueue_native_command`. I also like `enqueue_native_async_command` just so that it stresses to the user that only async work should be submitted in the callable, but this might be somewhat verbose. Let me know what you think

Reply:
+1 to `enqueue_native_command`

Reply:
I've change all naming to `enqueue_native_command`

Reply:
Please also:

* Rename the feature test macro
* Rename the file which is currently named "sycl_ext_oneapi_enqueue_custom_operation.asciidoc".

Reply:
Yes good shout. Thanks.

Reply:
Have done so